docs/book/v6/tutorials/create-book-module.md

```diff
@@ -88,47 +136,110 @@ class ConfigProvider
     {
         return [
             'dependencies'     => $this->getDependencies(),
-            'doctrine'     => $this->getDoctrineConfig(),
-             MetadataMap::class => $this->getHalConfig(),
+            MetadataMap::class => $this->getHalConfig(),
         ];
     }
 
     private function getDependencies(): array
     {
         return [
             'delegators' => [
-                Application::class => [
-                    RoutesDelegator::class
-                ]
+                Application::class              => [RoutesDelegator::class],
+                PostBookHandler::class          => [HandlerDelegatorFactory::class],
+                GetBookHandler::class           => [HandlerDelegatorFactory::class],
+                GetBookCollectionHandler::class => [HandlerDelegatorFactory::class],
             ],
-            'factories' => [
+            'factories'  => [
+                PostBookHandler::class          => AttributedServiceFactory::class,
+                GetBookHandler::class           => AttributedServiceFactory::class,
+                GetBookCollectionHandler::class => AttributedServiceFactory::class,
+                BookService::class              => AttributedServiceFactory::class,
             ],
-            'aliases'   => [
+            'aliases'    => [
+                BookServiceInterface::class => BookService::class,
             ],
         ];
     }
 
-    private function getDoctrineConfig(): array
+    private function getHalConfig(): array
     {
         return [
-            
+            AppConfigProvider::getResource(Book::class, 'book::view-book'),
+            AppConfigProvider::getCollection(BookCollection::class, 'book::list-books', 'books'),
         ];
     }
+}
+```
 
-    private function getHalConfig(): array
+* `src/Core/src/Book/src/ConfigProvider.php`
+
+In `src/Core/src/Book/src` we will create 1 PHP file: `ConfigProvider.php`. This file contains the necessary configuration for Doctrine ORM.
+
+```php
+<?php
+
+declare(strict_types=1);
+
+namespace Api\Book;
+
+use Api\App\ConfigProvider as AppConfigProvider;
+use Api\App\Factory\HandlerDelegatorFactory;
+use Api\Book\Collection\BookCollection;
+use Api\Book\Handler\GetBookCollectionHandler;
+use Api\Book\Handler\GetBookHandler;
+use Api\Book\Handler\PostBookHandler;
+use Api\Book\Service\BookService;
+use Api\Book\Service\BookServiceInterface;
+use Core\Book\Entity\Book;
+use Dot\DependencyInjection\Factory\AttributedServiceFactory;
+use Mezzio\Application;
+use Mezzio\Hal\Metadata\MetadataMap;
+
+class ConfigProvider
```

Comment:
This ConfigProvider file needs to be redone from scratch because it references items from a non-Core ConfigProvider.